### DISCUSSION TITLE
What is the difference between `octocrab::models::events::payload::EventPayload` and `octocrab::models::webhook_events::payload::WebhookEventPayload`?

### DISCUSSION THREAD

**Discussion by InAnYan** [XAMPPRocky/octocrab#691] [Q&A] [answered]
Seems to be very similar...

**Comment by manchicken** [comment, marked as answer]
If I'm not mistaken, `octocrab::models::events::payload::EventPayload` is for the `/events` API endpoints, and the `octocrab::models::webhook_events::payload::WebhookEventPayload` type is for deserializing incoming webhook events if you're writing a GitHub App that is triggered by webhooks.

Not certain, but that's kinda what the code looks like to me.

(I know that this is an old question, but it looks like one that others might also have)
